src/components/ButtonBox/index.jsx

```diff
@@ -21,10 +22,14 @@ const ButtonBox = ({
   if (previousPath !== undefined) {
     buttonBox = (
       <div className={styles.next_and_prev_button}>
-        <Button type={PreviousButtonType} path={previousPath}>
+        <Button
+          type={PreviousButtonType}
+          path={previousPath}
+          isFormValid={true}
+        >
           {PreviousButtonText}
         </Button>
-        <Button type={nextButtonType} path={nextPath}>
+        <Button type={nextButtonType} path={nextPath} isFormValid={isFormValid}>
           {nextButtonText}
         </Button>
       </div>
```

Comment:
Something simpler like this maybe? (this is sudo code)
```jsx
return (
     <div className={styles.next_and_prev_button}>
        {previousPath && (
           <Button
            type={PreviousButtonType}
            path={previousPath}
            isFormValid={true}
          >
            {PreviousButtonText}
          </Button> 
        )}
        <Button type={nextButtonType} path={nextPath} isFormValid={isFormValid}>
          {nextButtonText}
        </Button>
      </div>
)
```

Reply:
done